helm_drydock/drivers/__init__.py

```diff
@@ -11,11 +11,92 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Almost never a good idea to fill `__init__.py` with anything other then import helpers - either something like `/drivers/driver.py` or `/drivers/base.py` if there is an anticipation of more files or if not, simply `drivers.py`